sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1774,6 +1775,34 @@ class Analyzer(
   }
 
   /**
+   * Substitute Hints.
+   */
+  object SubstituteHint extends Rule[LogicalPlan] {
+    def apply(plan: LogicalPlan): LogicalPlan = plan transform {
```

Comment:
Oh, @hvanhovell .
Your question is not only about `alias` and `nesting`, but also about `LEFT ANTI JOIN`, right?
I tried `explain ~~`, and found it fails on Hive.